.github/workflows/django_check.yaml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+
+jobs:
+  ruff:
+    runs-on: ubuntu-latest
+    name: Check Django
+    steps:
+      - name: "Checkout Repo"
+        uses: actions/checkout@v4
+      - name: Check Django
+        uses: dagger/dagger-for-github@v5
+        with:
+          verb: call 
+          args: "linters check_django"
```

Comment:
I'm pretty sure this should be `linters check-django` (dash instead of underscore).